apps/bttester/syscfg.yml

```diff
@@ -145,3 +145,38 @@ syscfg.vals:
 
     BLE_MESH_ADV_BUF_COUNT: 20
     BLE_MESH_TX_SEG_MAX: 6
+
+    BLE_VERSION: 54
+    BLE_ISO: 1
+    BLE_ISO_BROADCASTER: 1
+    BLE_MAX_BIG: 1
+    BLE_MAX_BIS: 1
+    BLE_EXT_ADV: 1
+    BLE_PERIODIC_ADV: 1
+    BLE_EXT_ADV_MAX_SIZE: 261
+    BLE_MULTI_ADV_INSTANCES: 2
+    MSYS_1_BLOCK_COUNT: 32
+    BLE_PHY_2M: 1
+
+syscfg.defs:
```

Comment:
syscfg.defs: are duplicated. Please move this up to existing ones

Reply:
Done